packages/poseidon/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@zk-kit/poseidon-artifacts",
   "description": "zk-kit poseidon artifacts",
   "license": "MIT",
-  "version": "1.0.0-beta.1",
+  "version": "1.0.0-beta.4",
```

Comment:
to trigger ci and match https://github.com/privacy-scaling-explorations/zk-kit/blob/687bddcb2a124b478af278729f44dc7632eb34d6/packages/poseidon-proof/package.json#L3